lib/src/view/settings/settings_screen.dart

```diff
@@ -456,3 +493,55 @@ class _Body extends ConsumerWidget {
     }
   }
 }
+
+class _SettingsSliderDialog extends StatefulWidget {
+  const _SettingsSliderDialog({
+    required this.title,
+    required this.value,
+    required this.values,
+    required this.onChanged,
+  });
+
+  final String title;
+  final num value;
+  final List<num> values;
+  final void Function(num value) onChanged;
+
+  @override
+  _SettingsSliderDialogState createState() => _SettingsSliderDialogState();
+}
+
+class _SettingsSliderDialogState extends State<_SettingsSliderDialog> {
+  late num value = widget.value;
+
+  @override
+  Widget build(BuildContext context) {
+    return AlertDialog(
+      title: Text(widget.title),
+      content: StatefulBuilder(
+        builder: (context, setState) {
+          return Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              NonLinearSlider(
```

Comment:
I think you could use flutter's `Slider.adaptive` instead since that one is linear.

Reply:
Fixed